great_expectations/validator/validator.py

```diff
@@ -627,11 +628,8 @@ def _build_rule_based_profiler_from_config_and_runtime_args(
         override_profiler_config.pop("config_version", None)
 
         override_variables: Optional[Dict[str, Any]] = override_profiler_config.get(
-            "variables"
+            "variables", {}
```

Comment:
Please update this type hint 😄

Reply:
@cdkini Removing `Optional` if that is what you spotted.  Thanks.